### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by t-richards**
The path to Windows 10 Privacy is going to be long and arduous. Why not start it off right? :smile: 

Some notable changes:
- Renamed default namespace from `Windows_10_Privacy` to `WindowsTenPrivacy`
- Renamed some form controls to have more meaningful names (e.g. `richTextBox1` => `rchtxtLogMessages`)
- Changed targeted runtime for project/solution to `.NET 4.6`
- Added [NLog](https://github.com/NLog/NLog) for logging
- Put form controls in a `TableLayoutPanel` for resizability

Things that might require explanation:
- Automatic "checkbox is checked" processing happens by creating a `PrivacyComponent.<checkbox_tag_here>` class and calling the `.Remove()` method on it

**TL;DR:** Took all the methods from `Form1.cs` and put them in classes.

### CODE REVIEW

**Comment by virustotalop** [comment]
Thank you for the refactor, I have meaning to do that myself but I am a bit inexperienced when it comes to C#.
